views/forms/self_referral_form.php

```diff
@@ -41,68 +41,66 @@
                     $self_pers_street_num = $self_address_info[$i];
                 }
             } else {
-                $self_pers_street_name = "";
                 $self_pers_street_name .= $self_address_info[$i] . " ";
             }
         }
 
     $self_pers_zip = !empty($_POST['self_pers_zip']) ? $_POST['self_pers_zip'] : NULL;
     $self_pers_state = !empty($_POST['self_pers_state']) ? $_POST['self_pers_state'] : NULL;
     $self_pers_city = !empty($_POST['self_pers_city']) ? trim($_POST['self_pers_city']) : NULL;
-    $self_apt_info = !empty($_POST['self_apt_info']) ? trim($_POST['self_apt_info']) : NULL;
+    $self_apt_info = !empty($_POST['self_apt_info']) ? trim($_POST['self_apt_info']) : "";
     $self_pers_phone = !empty($_POST['self_pers_phone']) ? $_POST['self_pers_phone'] : NULL;
 
     // Second Card (Additional Information)
-    $self_involvement = !empty($_POST['self_involvement']) ? $_POST['self_involvement'] : NULL;
+    $self_involvement = !empty($_POST['self_involvement']) ? $_POST['self_involvement'] : 0; // TODO: Fix no entry.
```

Comment:
If no entry means, user did not select a radio button "Yes"/"No", you can make it required to respond.

http://jsfiddle.net/2du3s/82/

only one needs the `required` attribute within the `<form>`

Reply:
I thought we might have to enter a NULL if the user doesn't select either (yes or no), that's what I meant.

Reply:
I might be missing some context, but it seems like a yes/no question should be required.

Reply:
I'm with you haha, but I was under the impression that TECHNICALLY a user could submit this form with ONLY their first and last name (as those are the only required field for the form to submit).